dagshub/auth/oauth.py

```diff
@@ -15,6 +15,18 @@ def oauth_flow(
     host: str,
     client_id: Optional[str] = None
 ) -> Dict:
+    
+    """
+    Initiate the OAuth 2.0 flow for obtaining an access token.
```

Comment:
@kbolashev do we actually want people to use this function? I'm worried we might document functions that we don't want people using on a regular basis.

Should we add something that points to the correct function to be used (`get_token`)?

Reply:
This is an internal function, isn't supposed to be seen by users.
Don't see anything bad in having a docstring on it though

Reply:
@Nikitha-Narendra I think we shouldn't have this documented for now. Can you remove this?